packages/server/src/server.js

```diff
@@ -31,7 +31,7 @@ const {
 
 const { ALGOLIA_INDEX_NAME } = process.env;
 
-if (dev && /heroku_.*/.test(DATABASE_NAME)) {
+if (dev && /heroku_23.*/.test(DATABASE_NAME)) {
```

Comment:
So this is the result of the production database name changing, yes?

Reply:
nope its the result of the dev database changing... which makes me think nobody has actually used the dev database yet